.github/workflows/stlab.yml

```diff
@@ -53,7 +56,8 @@ jobs:
           python.exe --version
           cmake --version
           python.exe -m pip install --upgrade pip
-          pip.exe install conan_package_tools==0.37.0
+          pip.exe install markupsafe==2.0.1
```

Comment:
See https://github.com/conan-io/conan/issues/10611. I could not find a more appropriate workaround than this. Conan will fail if this dependency is not available.